src/main/groovy/grails/plugin/cache/web/filter/PageFragmentCachingFilter.java

```diff
@@ -175,21 +174,25 @@ public void setUrlMappingsHandlerMapping(UrlMappingsHandlerMapping urlMappingsHa
 
 	@Override
 	protected void doFilter(HttpServletRequest request, HttpServletResponse response, FilterChain chain) throws Exception {
-		// TODO need blocking cache stuff from CachingFilter
-
-		if(urlMappingsHandlerMapping != null) {
-			// initialize the state for the mapping
-			urlMappingsHandlerMapping.getHandler(request);
+		// todo: need blocking cache stuff from CachingFilter
+		try {
```

Comment:
Adding try catch on get handler as this is *probably* what is causing the stack overflow in tomcat on spring boot when %% used